attributes/conditions.rb

```diff
@@ -20,3 +20,4 @@
 default['conditions']['intel_hpc_platform_supported'] = !arm_instance? && platform_supports_intel_hpc_platform?
 default['conditions']['dcv_supported'] = platform_supports_dcv?
 default['conditions']['ami_bootstrapped'] = ami_bootstrapped?
+default['conditions']['efa_supported'] = (!arm_instance?) || (node['cfncluster']['cfn_base_os'] != "centos8")
```

Comment:
try to run rubocop on this change, it seems to me that you can remove some extra parenthesis.

Reply:
done